eng/pipelines/templates/stages/archetype-python-release.yml

```diff
@@ -175,7 +177,8 @@ stages:
                               SparseCheckoutPaths:
                                 - docs-ref-services/
                                 - metadata/
-
+                              PackageSourceOverride: ${{parameters.PackageSourceOverride}}
```

Comment:
Given that we've just published to PyPI do we need to override the package source here?